pubsublite-spark-sql-streaming/pom.xml

```diff
@@ -19,24 +19,28 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <encoding>UTF-8</encoding>
     <scala.version>2.11.12</scala.version>
+    <scala.version.short>2.11</scala.version.short>
+    <spark.version>2.4.7</spark.version>
+    <grpc.netty.version>1.30.2</grpc.netty.version>
+    <netty.tcnative.version>2.0.34.Final</netty.tcnative.version>
```

Comment:
Are these versions used anywhere? If you are using `grpc-netty-shaded` than probably you do not need to set these explicitly.

Reply:
Removed.